resources/api.rb

```diff
@@ -3,7 +3,11 @@
 property :args, [Hash, String, NilClass], desired_state: false
 property :endpoint, String, desired_state: false, identity: true, default: 'http://localhost:8081/service/siesta/rest/v1/script/'.freeze
 property :username, String, desired_state: false, identity: true, default: 'admin'.freeze
-property :password, String, desired_state: false, identity: true, default: 'admin123'.freeze
+if ::Gem::Requirement.new('>= 12.14.34').satisfied_by?(::Gem::Version.new(::Chef::VERSION))
```

Comment:
OPT: you can increase the chef version requirement in metadata.rb and support only chef >= 12.14.34